docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -146,6 +146,22 @@ or
 The floating label is absolutely positioned, it won't impact the layout of the page.
 You need to make sure that the input is larger than the label to display correctly.
 
+### Number inputs allowing non-numeric characters
+
+By specification, inputs of type "number" allow the character 'e' (and other characters such as '+', '-', '.') to be entered.
```

Comment:
This is only one of the 4 issues, I think that we could mention them all, with a link for more details to https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/.